timemachine/fe/single_topology.py

```diff
@@ -1251,21 +1261,19 @@ def get_other_idxs():
             get_lig_idxs(),
             get_water_idxs(),
             get_other_idxs(),
-            host_params,
+            host_nonbonded.params,
             guest_ixn_water_params,
             guest_ixn_other_params,
-            beta=beta,
+            beta=host_nonbonded.potential.beta,
             cutoff=cutoff,
         )
 
-        hg_total_pot = [combined_nonbonded] + ixn_pots
-        hg_total_params = [hg_nb_params] + ixn_params
-        sum_pot = SummedPotential(hg_total_pot, hg_total_params)
```

Comment:
Are there any performance considerations for this change?

Reply:
I've verified by profiling `test_relative_free_energy` that there is no significant performance impact.